2.0/controllers.md

```diff
@@ -42,7 +42,7 @@ Route.get('/users', 'UserController.index')
 
 ## Resourceful Controllers
 
-Resourceful routes define multiple routes and attach conventional methods to them under single route definition.
+[Resourceful routes](http://adonisjs.com/docs/2.0/routing#restful-routes) define multiple routes and attach conventional methods to them under single route definition.
```

Comment:
It's in the develop branch for the moment.

https://github.com/adonisjs/docs/blob/develop/2.0/routing.md#restful-routes